istioctl/cmd/proxyconfig.go

```diff
@@ -518,6 +518,7 @@ func allConfigCmd() *cobra.Command {
 						return err
 					}
 				}
+				configWriter.IncludeConfigType = true
```

Comment:
could this be decided by feature gate flag or environment variable?

Reply:
You mean adding a flag to control this behavior? I think this should only exist in `pc all` since different config dumps are listed. For single config we may not need this, like the difference between `kubectl get pod` and `kubectl get all`. This is to exactly mimic how k8s works.

Reply:
Yeah,  I understand it, but if this `IncludeConfigType` is aways `true`, what's the meaning of its existence?

Reply:
I see, thanks for the explaination. I'll revise it